vendor/init.bat

```diff
@@ -266,7 +266,7 @@ if not defined git_locale if exist "%GIT_INSTALL_ROOT%\usr\bin\env.exe" set git_
 if not defined git_locale for /F "tokens=* delims=" %%F in ('where env.exe 2^>nul') do ( if not defined git_locale  set git_locale="%%F" /usr/bin/locale )
 
 if defined git_locale (
-  %lib_console% debug_output init.bat "Env Var - git_locale=%git_locale%"
+  %lib_console% debug_output init.bat "Env Var - git_locale="%git_locale%""
```

Comment:
I don't understand why this is needed. `"Env Var - git_locale=%git_locale%"` is a message to be echoed to screen quoting git local inside this message seems unnecessary and could break things.

Reply:
Without the quotes I get the following error.

![after-lib_path-change](https://user-images.githubusercontent.com/4619155/90201704-035cfe80-de1f-11ea-9dd6-52c62b9756df.png)